### DISCUSSION TITLE
It's possible to send mqtt commands to other zigbee devices in the same network?

### DISCUSSION THREAD

**Discussion by LeeWiu** [arendst/Tasmota#22106] [Q&A]
Hi, I want to send mqtt command "toggle livingroom_lamp" from a tasmota button when double press, it's possible?

**Comment by Pietjuhbel** [comment]
You can attach a rule on the button press and then have the rule send the mqtt command.

**Reply by LeeWiu** [reply]
Yes, I want to create a rule, but what is the exactly command?
